pkg/ingester/client/streaming.go

```diff
@@ -90,6 +90,7 @@ func (s *SeriesChunksStreamReader) StartBuffering() {
 		onError := func(err error) {
 			s.errorChan <- err
 			log.Error(err)
```

Comment:
can we replace this with `ext.Error.Set(log.Span, true)`

Reply:
Yep, that's a nicer solution, thanks.